Task_TheFlexGame/index.css

```diff
@@ -1 +1,61 @@
 /* Start coding below for FLEX GAME TASK*/
+
+.main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.header {
+  background-color: red;
+  padding-bottom: 20px;
+  padding-left: 10px;
+  padding-top: 10px;
+  height: 100%;
+  flex: 1 1 0;
+  width: 460px;
+}
+
+.content {
+  display: flex;
+  height: 100%;
+  width: 470px;
+}
+.menu {
+flex-basis: 20%;
+display: flex;
+flex-direction: column;
+gap: 10px; 
+background-color: purple;
+height: 400px;
+padding: 30px;
+}
+
+.menuButton {
+  text-align: center;
+  background-color: aliceblue;
+  padding: 10px;
+
+
+}
+.contentContainer {
+  flex-basis: 60%;
+  background-color: green;
+  
+}
+
+.infoContainer {
+  flex-basis: 20%;
+  background-color: cyan;
```

Comment:
The 'width' property on line 49 should not be set for a flex item. It should be set on the parent container instead.